authorizer.go

```diff
@@ -310,6 +334,17 @@ func readPublicKey(cert []byte) (*rsa.PublicKey, error) {
 	return publicKey, nil
 }
 
+func isApiKey(headers http.Header) bool {
+	apiKey := headers.Get(ApiKeyHeader)
+	if apiKey == "" {
+		return false
+	}
+	if apiKey != os.Getenv("BACKEND_API_KEY") {
```

Comment:
Don't get the value from env, pass it as a parameter on the middleware

Reply:
ok